build/Linux+BSD/portable/copy-libs

```diff
@@ -93,6 +93,8 @@ getCrossPlatformDependencies() {
 
   copyQt plugins/sqldrivers/libqsqlite.so
 
+  copyQt plugins/xcbglintegrations/libqxcb-glx-integration.so
```

Comment:
I don't see a Windows equivalent for this, no library of that or a similar name is part of Qt for Windows.